packages/terra-form-input/src/terra-dev-site/doc/form-input/InputField.2.doc.mdx

```diff
@@ -0,0 +1,50 @@
+import { Badge } from 'terra-form-input/package.json?dev-site-package';
+
+import InputFieldExample from '../example/InputField?dev-site-example';
+import RequiredInputFieldExample from '../example/RequiredInputField?dev-site-example';
+import IncompleteInputFieldExample from '../example/IncompleteInputField?dev-site-example';
+import NumberInputFieldExample from '../example/NumberInputField?dev-site-example';
+
+
+import FormInputFieldPropsTable from 'terra-form-input/src/InputField?dev-site-props-table';
+
+<Badge />
+
+# Terra Form Input Field
+
+[terra-form-input](https://github.com/cerner/terra-core/tree/master/packages/terra-form-input) component that is wrapped inside a
+[terra-form-field](https://github.com/cerner/terra-core/tree/master/packages/terra-form-field) component.
+
+## Getting Started
+
+- Install with [npmjs](https://www.npmjs.com):
+  - `npm install terra-form-input`
+
+## Usage
+
+```jsx]
+import InputField from 'terra-form-input/lib/InputField';\
```

Comment:
There's a couple syntax issues here. 

1. The markdown back ticks for the jsx declaration has a bracket. ```jsx]
2. The trailing back slash in the import \

Reply:
Removed here https://github.com/cerner/terra-core/pull/2830/commits/631d0f64c7f790d95ee1f881a888aab064203d47.